go/libraries/doltcore/schema/typeinfo/varstring.go

```diff
@@ -83,17 +83,9 @@ func CreateVarStringTypeFromParams(params map[string]string) (TypeInfo, error) {
 // ConvertNomsValueToValue implements TypeInfo interface.
 func (ti *varStringType) ConvertNomsValueToValue(v types.Value) (interface{}, error) {
 	if val, ok := v.(types.String); ok {
-		strVal, err := ti.sqlStringType.Convert(string(val))
```

Comment:
Profiling these changes, but looks good overall.  This line was the worst performer, so as long as it's gone I imagine this should be fine.